eskip/eskip.go

```diff
@@ -335,6 +336,9 @@ type Route struct {
 
 	// Namespace is deprecated and not used.
 	Namespace string
+
+	// TLS Certificate to use for terminating TLS on the route
+	TLSCert tls.Certificate
```

Comment:
On every routing table update cycle dataclient uses string route representation to figure out if route has changed. In the current implementation afaict the secret would be reloaded on every cluster state update and hence `tls.Certificate` will change but it is not reflected in the string representation of the route and therefore route will not be considered as updated.

Reply:
Resolving this since not using routes to determine TLS certificates